### ISSUE TITLE
trying out the new service metadata

### ISSUE THREAD

**Issue by jdanyow** [MicrosoftDocs/SamplesPages#96]
this github issue should have a "Service" bullet point.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8d48ec42-ecba-8e3c-cc6a-fa25ade938cd
* Version Independent ID: ea115f4c-19ef-d80c-1e0a-dcbdcc0770f5
* [Content](https://review.docs.microsoft.com/en-us/devsandbox/conceptual/azure-cli?branch=master)
* [Content Source](https://github.com/MicrosoftDocs/DevSandbox/blob/master/devsandbox/conceptual/azure-cli.md)
* Service: multiple

**Comment by EisenbergEffect** [comment]
Why does it say "multiple"?

**Comment by jdanyow** [comment]
the [yaml header](https://github.com/MicrosoftDocs/DevSandbox/blob/master/devsandbox/conceptual/azure-cli.md) for this document has `ms.service: multiple`.

**Comment by jdanyow** [comment]
while you're here, let me run this by you....

The current implementation looks for the `ms.service` meta. If it's not found, it uses the text "unspecified".  It also lower-cases the `ms.service` value. I was thinking this would make the querying more consistent and enable querying issues for docs that do not have a service specified. Thoughts?

**Comment by EisenbergEffect** [comment]
Cool. Just making sure there weren't multiple values and that we were doing some funky thing there. Thanks!

**Comment by EisenbergEffect** [comment]
Yes, that sounds pretty good. Also, you need to look for ms.product I think. Some content has that instead of service, but they are basically the same thing.

**Comment by jdanyow** [comment]
I looked at the feedback we currently have using this query:

https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+created%3A%3E%3D2017-12-15+is%3Apublic+in%3Abody+%22Version+Independent+ID%22+-repo%3AMicrosoftDocs%2FSamplesPages+-repo%3Ajdanyow%2Ftest2

I didn't find any sign of `ms.product` so I held off on that for now. I can add it anytime.

**Comment by EisenbergEffect** [comment]
Whether or not you see ms.product or not will depend entirely on which docset you are looking at. Some use service and some use product. There’s been this split since before I joined, so we need to account for both for now, but we can all it “Service” in the issue, unless Sarah says to call it something else. That’s something to ask her about.

--
Rob Eisenberg
Sr. PM Manager – APEX Online Experience Platform

**Comment by jdanyow** [comment]
sounds good, I'll add it now, if it needs to change, let me know, no problem
